adapter/main.go

```diff
@@ -209,6 +211,7 @@ func main() {
 	cmd.Flags().StringVar(&prometheusMetricsPath, "metrics-path", "/metrics", "Set the path for the prometheus metrics endpoint")
 	cmd.Flags().Float32Var(&adapterClientRequestQPS, "kube-api-qps", 20.0, "Set the QPS rate for throttling requests sent to the apiserver")
 	cmd.Flags().IntVar(&adapterClientRequestBurst, "kube-api-burst", 30, "Set the burst for throttling requests sent to the apiserver")
+	cmd.Flags().BoolVar(&disableCompression, "disable-compression", false, "Disable response compression for k8s restAPI in client-go. ")
```

Comment:
Actually, I think we should make this `true` by default for both Operator and Metrics Server

Reply:
@kedacore/keda-core-contributors WDYT?

Reply:
Let's try it out, we can always revert, @ckuduvalli could you please change the default value? We will run e2e test and if it is good, we will merge it (and update docs PR as well) Thanks 🙏

Reply:
have changed the default value to true

Reply:
have updated charts as well
https://github.com/kedacore/charts/pull/326